### DISCUSSION TITLE
limit the maximum power  of cpu

### DISCUSSION THREAD

**Discussion by Bitcion** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1116] [Q&A]
Aidente can identify the real-time power consumption of the CPU, which reminds me of a volta for mac application that can directly limit the maximum power of the CPU. Can I manually limit the maximum power of the CPU in battery mode to extend the battery life?

**Comment by MatthiasKerbl** [comment]
Hi @Bitcion,

The only way this is possible on newer MacBooks to my knowledge is with the macOS Low Power Mode.

Best,
Matthias
